resources/definitions/ultimaker_method_base.def.json

```diff
@@ -412,9 +416,10 @@
         "support_roof_height": { "value": "4*layer_height if extruder_nr == support_extruder_nr else 5*layer_height" },
         "support_roof_material_flow": { "value": "material_flow" },
         "support_supported_skin_fan_speed": { "value": "cool_fan_speed_max" },
-        "support_top_distance": { "value": "support_z_distance" },
+        "support_top_distance": { "value": "0.25 if extruder_nr == support_extruder_nr else support_z_distance" },
```

Comment:
This is not required. The support_top_distance is limited to the support_roof_extruder_nr, so there is no need to use a formula here.

If you do want to use a formula, the current one is incorrect; It should use the support_roof_extruder_nr instead

Reply:
Aka; For all practical intents and purposes it will be 0.25, so you can simplify it